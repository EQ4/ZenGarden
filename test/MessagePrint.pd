#N canvas 720 303 450 300 10;
#X obj 32 19 loadbang;
#X obj 32 126 print abc;
#X obj 107 126 print 123;
#X obj 183 126 print;
#X connect 0 0 1 0;
#X connect 0 0 2 0;
#X connect 0 0 3 0;
