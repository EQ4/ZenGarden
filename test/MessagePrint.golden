[@ 0.000ms] abc: bang
[@ 0.000ms] print: bang
[@ 0.000ms] print: bang
